### PR TITLE
Project Library: library-feature

### PR DESCRIPTION
**Pull request by PhillipUg**
In the following project we have done the following:
 - built a simple library app with Javascript, HTML & CSS
 - we enabled users to add books, remove books and change their state from either read or unread
- we implemented a simple form to help users add books

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @PhillipUg and @SafaErden 
### You have done well so far :clap: However, you have an issue to fix:-
- [ ] Separate you dom from application logic
- [ ] You are also required to store the book object in an array, see the image below
array-book
### Make the change and submit for another code review.
- Happy coding!

By Raymond
Slack @iamraymond

File: app.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  const displayBooks = local.map((item) => {
+    if (item.read === true) {
+      checked = 'checked';
+    } else {
+      checked = '';
+    }
+    return `<div class="book-item" id=${item.id}><span class="bold"> Author: </span> ${item.author}  &nbsp;&nbsp;<span class="bold"> Title: </span> ${item.title} &nbsp;&nbsp;<span class="bold"> Pages:  </span> ${item.pages} &nbsp;&nbsp;<div class="status-wrapper"><input type="checkbox" id="status" name="status" class="status" ${checked}><label for="status">Read &nbsp; &nbsp;</label></div><button onclick="removeFromLocalStorageArray('books',${item.id})" class="remove-button btn btn-primary">Remove</button></div>`;
+  });
+  container.innerHTML = displayBooks.join('');
+}
```
Comment (rOluochKe) [inline app.js]: ### separate dom from the logic

File: app.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+const container = document.querySelector('.container');
+const addBook = document.getElementById('addBook');
+const form = document.querySelector('.hidden');
+const add = document.getElementById('add');
```
Comment (rOluochKe) [inline app.js]: ### Separate dom from logic

**Review by uche-inyama — APPROVED** [review]
# Approved

## Hi team :clap: 

This project is approved :100: 

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Pull request merged**